modules/system/defaults/finder.nix

```diff
@@ -2,6 +2,9 @@
 
 with lib;
 
+let
+  cfg = config.system.defaults.finder;
+in
```

Comment:
```suggestion
let
  inherit (lib) mkOption types ...;

  cfg = config.system.defaults.finder;
in
```

Reply:
Woah this is neat syntax, thanks for suggesting it